rawx-lib/src/attr_handler.c

```diff
@@ -168,6 +196,57 @@ _get (int fd, const char *k, gchar **pv)
 
 #define GET(K,R) _get(fd, ATTR_DOMAIN "." K, &(R))
 
+static GString*
+_search_user_oio(gchar* prefix, gchar * xattrlist, ssize_t size, gchar *name)
+{
+	GString* list_user = g_string_new("");
+	gchar* elem = NULL;
+	gint prefix_size = strlen(prefix) + 1;
+	for (gint i = 0; i < size; i+= strlen(&xattrlist[i]) + 1) {
+		elem = g_strrstr(&xattrlist[i], prefix);
+		if (elem) {
+			if (name)
+				elem = g_strrstr(&xattrlist[i], name);
```

Comment:
You should skip `prefix` and do a length-limited string comparison, or a substring may match unexpectedly.